File: src/main/frontend/src/app/common/contact-form.ts
export class ContactForm {

  constructor(private _firstName: String, private _lastName: String, private _addLine1: String,
              private _addLine2: String, private _city: String, private _state: String, private _zip: String,
              private _phone: String, private _email: String) {
  }

  set firstName(value: String) {
    this._firstName = value;
  }

  set lastName(value: String) {
    this._lastName = value;
  }

  set addLine1(value: String) {
    this._addLine1 = value;
  }

  set addLine2(value: String) {
    this._addLine2 = value;
  }

  set city(value: String) {
    this._city = value;
  }

  set state(value: String) {
    this._state = value;
  }

  set zip(value: String) {
    this._zip = value;
  }

  set phone(value: String) {
    this._phone = value;
  }

  set email(value: String) {
    this._email = value;
  }
}

File: src/main/java/org/_3rev/resume/valueobject/ContactForm.java
package org._3rev.resume.valueobject;

public class ContactForm {

    private String firstName;
    private String lastName;
    private String objective;
    private String addLine1;
    private String addLine2;
    private String city;
    private String state;
    private String zip;
    private String phone;
    private String email;

    public ContactForm() {
        super();
    }

    public ContactForm(String firstName, String lastName, String objective, String addLine1, String addLine2, String city, String state, String zip, String phone, String email) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.objective = objective;
        this.addLine1 = addLine1;
        this.addLine2 = addLine2;
        this.city = city;
        this.state = state;
        this.zip = zip;
        this.phone = phone;
        this.email = email;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getObjective() {
        return objective;
    }

    public void setObjective(String objective) {
        this.objective = objective;
    }

    public String getAddLine1() {
        return addLine1;
    }

    public void setAddLine1(String addLine1) {
        this.addLine1 = addLine1;
    }

    public String getAddLine2() {
        return addLine2;
    }

    public void setAddLine2(String addLine2) {
        this.addLine2 = addLine2;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getZip() {
        return zip;
    }

    public void setZip(String zip) {
        this.zip = zip;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}
